### PR TITLE
Register page

### LINKED ISSUE #25: Register page
All components currently exist.
Use React Native Paper for input.
Difficulty level: **2/5**

The issue is to fix the following: 

Screenshot 2020-04-15 at 13 15 22

### PR DESCRIPTION
**Pull request by eckernas**
closes #25

### CODE REVIEW

**Comment by eckernas** [comment]
Screen Shot 2020-04-27 at 10 02 57 PM

The logo gets cut off like this when I start typing. Adding zIndex: -1 in style dividedContent fixed it for me. So maybe we can keep that line? Or is there another better way to fix the problem?

**Comment by AdamJonsson** [comment]
@eckernas Ahh, sorry about that, will add it back 😄

File: src/pages/register-page.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                        maxLength={10}
+                        selectionColor={paperTheme.colors.onPrimary}
+                        keyboardType='phone-pad'
+                        value={numberInput}
+                        onChangeText={text => setNumber(text)}
+                        error={showError}
+                        style={styles.textInput}/>
+                    {showError ? <Text style={styles.styleErrorText}>You need to enter 10 digits.</Text> : <></>}
+                    <View style={styles.buttonContainer}>
+                        <Button onPress={checkPhoneNumber} color={paperTheme.colors.onPrimary}>Send registration code</Button>
```
Comment (AdamJonsson) [inline src/pages/register-page.tsx:77]:
```suggestion
                        <Button size='big' onPress={checkPhoneNumber} color={paperTheme.colors.onPrimary}>Send registration code</Button>
```

File: src/pages/register-page.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                    </View>
+                </View> 
+            </View>
+        </SafeAreaView>
+    )
+
+    return (
+        <TouchableWithoutFeedback onPress={Keyboard.dismiss}>
+            <DividedView upper={upper} lower={lower}/>
```
Comment (AdamJonsson) [inline src/pages/register-page.tsx:86]: For me, the divided view needs to be wrapped with a view in order to the `onPress={Keyboard.dimiss}` to work:
```suggestion
            <View style={{flex: 1}}>
                <DividedView upper={upper} lower={lower}/>
            </View>
```

**Comment by eckernas** [comment]
Ah nice, works for me as well! Is it okay to merge then?

**Pull request merged**